docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml

```diff
@@ -1215,16 +1215,17 @@ lock=/var/lock/HPCCSystems</programlisting>
       <sect2 id="Using_nice">
         <title>Using nice</title>
 
-        <para>For our purposes <emphasis>nice</emphasis> is the Linux program
-        which invokes scripts and utilities with specified priorities. The
-        priority assigned to a process indicates to the CPU to provide more or
-        less time than to other processes. With a nice value of -20 having the
-        highest priority, and a value of 19 as the lowest priority.</para>
+        <para>The HPCC platform supports <emphasis>nice</emphasis>-based
+        priorities. As nice is the Linux program which invokes scripts and
```

Comment:
"As nice is the Linux program which invokes scripts and utilities with specified priorities" <--  Is a sentence fragment. Remove the word "As" and it's fine.